### DISCUSSION TITLE
Blob 22 (cannot delete the storage)

### DISCUSSION THREAD

**Discussion by XOP** [vercel/community#5837] [Help] [answered]
### Summary

Hello Team!

I'm having a Blob 22 situation...  

I created and connected a Blob storage to a project, then during testing unexpectedly exceeded the 100Mb quota shortly (I really thought it's a limit for a day) and found myself in a limbo. I would like to delete the storage, but here's a catch:
- To delete the storage I first need to delete the blobs manually
- If I go to the dashboard blobs cannot be listed, because of the quota exceeded (I guess?) or some other error
- If I try to list and del the blobs via SDK, I hit an error that account is suspended

The only thing that I've managed to do is disconnect the storage from the project.

Same cases I believe:  
https://github.com/orgs/vercel/discussions/5331  
https://github.com/orgs/vercel/discussions/5227  
https://github.com/orgs/vercel/discussions/4997

### Steps to Reproduce

- create a project and attach a Blob storage
- upload some objects (I only used 2 images)
- make requests involving Blob storage until you hit 100Mb limit (on Hobby plan)
- go to Dashboard / Blob storage - blobs cannot be listed and some error messages show up
- try to list blobs via SDK - it returns an error

<img width="664" alt="image" src="https://github.com/vercel/community/assets/1815878/794273b9-eeca-49f8-835d-a2c8d23f1953">

<img width="460" alt="image" src="https://github.com/vercel/community/assets/1815878/a54b2208-b998-4af3-b848-7a41784d1f12">

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. We made a change that lets you view the list and delete blobs even after limits are exceeded. Please give it a try and let me know if it works for you

**Reply by BillOTei** [reply]
Hi, I had similar issue, deleted all blobs but still get the error message
![image](https://github.com/vercel/community/assets/2312874/ac6bdaa9-1c59-43c5-bdf8-3292a6193d6c)

**Reply by amyegan** [reply]
Hey @BillOTei. The original error was the message saying, "An error has occurred while fetching your blobs" when trying to access blob storage information after exceeding limits. The change we made lets you see the list and delete blobs even after exceeding limits. If you were able to delete blobs just now, then that change is working. 🙂 

It looks like you're highlighting the "Limits Exceeded" status as an error, but that seems accurate based on the monthly storage size usage displayed. You would need to wait for the limits to reset for your usage cycle to see that go away.

https://vercel.com/docs/storage/vercel-blob/usage-and-pricing

**Reply by BillOTei** [reply]
thanks for getting back! I actually had that limit exceeded more than a month ago and I thought it would be solved by now. 
I tried deleting the blob a while back too. Is there a way to refresh that?
Thanks
